rviz_visual_tests/package.xml

```diff
@@ -0,0 +1,31 @@
+<package format="2">
+  <name>rviz_visual_tests</name>
+  <version>0.0.1</version>
```

Comment:
This needs to match the version of the other packages in this repository, which is `2.0.0`:

https://github.com/ros2/rviz/blob/4aaa893cfc7140d921a73f33236d6cd2df2aca37/rviz2/package.xml#L3